Library/Formula/eg.rb

```diff
@@ -0,0 +1,40 @@
+class Eg < Formula
+  homepage "https://github.com/srsudar/eg"
+  url "https://github.com/srsudar/eg/archive/v0.0.3.tar.gz"
+  sha256 "c257e334f2a2e494f9b9791b92c1d3f7b00da3e2e7ec47f02d720342b466f968"
+
+  def install
+    if Dir.exist? "#{var}/examples"
+      Dir["eg/examples/*"].each do |example|
+        next if example.to_s == '.' || example.to_s == '..'
+        mv example, "#{var}/examples/"
+      end
+      rmdir "eg/examples"
+    else
+      var.install "eg/examples"
+    end
+
+    inreplace "eg/eg_config.py",
+              /DEFAULT_EXAMPLES_DIR = .+$/,
+              "DEFAULT_EXAMPLES_DIR = os.path.join('#{var}', 'examples')"
```

Comment:
I don't think it is needed. It's Homebrew-specific. Original is: `os.path.join(os.path.dirname(__file__), 'examples')`. That works fine by default (I'm assuming), but because I move the examples directory to var, I have to change the examples directory default setting to where I move that directory. (Really, the blank line above the inreplace here should be below it.)